ui/src/components/credentials/IssuanceMethodForm.tsx

```diff
@@ -105,6 +95,7 @@ export function IssuanceMethodForm({
           ) {
             setIssuanceMethod({ ...allValues, linkExpirationTime: undefined });
           } else {
+            onChangeDid(changedValues);
```

Comment:
This callback should expect a valid `did` of type `string` and be called only after the values of the form have been parsed here (instead of having to parse in `IssueCredential.tsx` whatever value is changed in the form, as now).

Reply:
Updated one second ago